pkgs/tools/package-management/deploy-rs/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, lib, fetchFromGitHub, rustPlatform, CoreServices, SystemConfiguration }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "deploy-rs-unstable";
+  version = "2022-05-26";
```

Comment:
```
  pname = "deploy-rs";
  version = "unstable-2022-05-26";
```
oh, whatever

Reply:
Thanks, I'll make sure to fix this upon the next version bump